AppOne/AppOne/View/HomeVC.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  HomeVC.swift
+//  AppOne
+//
+//  Created by Asapu, Ajaikumar on 19.11.22.
+//
+
+import UIKit
+
+class HomeVC: UIViewController {
+    private var brandsCollectionView: UICollectionView!
+    private var dataSource: UICollectionViewDiffableDataSource<SectionTitles, Brands>!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureCollectionView()
+        configureDataSource()
+    }
+
+    func configureCollectionView() {
+        brandsCollectionView = UICollectionView(frame: view.bounds, collectionViewLayout: configureLayout())
+        brandsCollectionView.register(UICollectionViewCell.self, forCellWithReuseIdentifier: "Cell")
```

Comment:
this is not needed there is a `UICollectionView.CellRegistration`. This is the recommended way to go.

Reply:
Ok